pynamodb_attributes/timestamp.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime, timezone
+
+import pynamodb.constants
+from pynamodb.attributes import Attribute
+
+
+class TimestampAttribute(Attribute):
+    """"
+    Stores times as Unix epoch timestamps in a DynamoDB number.
+
+    >>> class MyModel(Model):
+    >>>   created_at_seconds = TimestampAttribute(default=lambda: datetime.now(tz=timezone.utc))
+    >>>   created_at_ms = TimestampMsAttribute(default=lambda: datetime.now(tz=timezone.utc))
+    """
+    attr_type = pynamodb.constants.NUMBER
+    _multiplier = 1.0
+
+    def deserialize(self, value: str) -> datetime:
+        return datetime.fromtimestamp(int(value) / self._multiplier, tz=timezone.utc)
+
+    def serialize(self, value: datetime) -> str:
+        return str(int(value.timestamp() * self._multiplier))
+
+    def __set__(self, instance, value: datetime) -> None:
+        if not isinstance(value, datetime):
+            raise TypeError(f"value has invalid type '{type(value)}; datetime expected")
+        if value.tzinfo is None:
+            raise TypeError("offset-aware datetime expected")
```

Comment:
I think most documentation uses "aware" or "timezone aware" to describe this situation

Reply:
Alright, will change to "aware".

I was going off:
```
>>> from datetime import datetime, timezone
>>> datetime.now(tz=timezone.utc) < datetime.utcnow()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can't compare offset-naive and offset-aware datetimes
```

Reply:
amazingly inconsistent I love it